packages/@aws-cdk/aws-eks/test/integ.eks-service-account-sdk-call.ts

```diff
@@ -0,0 +1,66 @@
+import * as path from 'path';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as ecrAssets from '@aws-cdk/aws-ecr-assets';
+import * as iam from '@aws-cdk/aws-iam';
+import { App, Stack, CfnOutput } from '@aws-cdk/core';
+import * as integ from '@aws-cdk/integ-tests';
+import * as cdk8s from 'cdk8s';
+import * as kplus from 'cdk8s-plus-21';
```

Comment:
NIT: Can this be updated to `cdk8s-plus-25`?

Reply:
EKS only supports up to Kubernetes 1.21: https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-eks/lib/cluster.ts#L802

I think we should use the latest `cdk8s-plus` that is supported by EKS, which also keeps this test consistent with the other integ tests in this module.

Reply:
discussed offline with eli, we can leave this at 21.